File: experiments/adrianco/experiment-5/runs/language=java_model=claude-opus-4-7_tooling=none/rep3/src/main/java/com/soccer/mcp/model/Player.java
package com.soccer.mcp.model;

public final class Player {
    private final String id;
    private final String name;
    private final Integer age;
    private final String nationality;
    private final Integer overall;
    private final Integer potential;
    private final String club;
    private final String position;
    private final String jerseyNumber;
    private final String height;
    private final String weight;
    private final String preferredFoot;

    public Player(String id, String name, Integer age, String nationality,
                  Integer overall, Integer potential, String club, String position,
                  String jerseyNumber, String height, String weight, String preferredFoot) {
        this.id = id;
        this.name = name;
        this.age = age;
        this.nationality = nationality;
        this.overall = overall;
        this.potential = potential;
        this.club = club;
        this.position = position;
        this.jerseyNumber = jerseyNumber;
        this.height = height;
        this.weight = weight;
        this.preferredFoot = preferredFoot;
    }

    public String id() { return id; }
    public String name() { return name; }
    public Integer age() { return age; }
    public String nationality() { return nationality; }
    public Integer overall() { return overall; }
    public Integer potential() { return potential; }
    public String club() { return club; }
    public String position() { return position; }
    public String jerseyNumber() { return jerseyNumber; }
    public String height() { return height; }
    public String weight() { return weight; }
    public String preferredFoot() { return preferredFoot; }
}

File: experiments/adrianco/experiment-48-fable5-gaps/brazil/runs/language=elixir_model=claude-fable-5_prompt=neutral/rep1/lib/brazilian_soccer_mcp/player.ex
defmodule BrazilianSoccerMcp.Player do
  @moduledoc """
  A player record from the FIFA dataset. `:name_search` and `:club_search`
  hold accent-stripped lowercase forms used for matching.
  """

  defstruct [
    :id,
    :name,
    :name_search,
    :age,
    :nationality,
    :overall,
    :potential,
    :club,
    :club_search,
    :position,
    :jersey_number,
    :height,
    :weight,
    :value,
    :wage,
    :preferred_foot,
    skills: %{}
  ]

  @forwards ~w(ST CF LW RW LS RS LF RF)
  @midfielders ~w(CM CDM CAM LM RM LCM RCM LDM RDM LAM RAM)
  @defenders ~w(CB LB RB LCB RCB LWB RWB)

  @doc """
  Expand a position query ("forward", "midfielder", "defender", "goalkeeper",
  or an exact code like "GK"/"ST") into the list of position codes it covers.
  """
  def position_codes(query) do
    case query |> to_string() |> String.downcase() |> String.trim() do
      q when q in ["forward", "forwards", "attacker", "striker", "atacante"] -> @forwards
      q when q in ["midfielder", "midfielders", "midfield", "meia"] -> @midfielders
      q when q in ["defender", "defenders", "defence", "defense", "zagueiro"] -> @defenders
      q when q in ["goalkeeper", "goalkeepers", "keeper", "gk", "goleiro"] -> ["GK"]
      q -> [String.upcase(q)]
    end
  end
end
